components/form/KeyValue.vue

```diff
@@ -300,7 +295,10 @@ export default {
     },
 
     containerStyle() {
-      return `grid-template-columns: repeat(${ 2 + this.extraColumns.length }, 1fr)${ this.removeAllowed ? ' 50px' : '' };`;
+      const gap = !this.isView ? '" 50px"' : '""';
```

Comment:
The triple quotes create a strange issue:
![triple](https://user-images.githubusercontent.com/40806497/152419983-071b1316-3d68-4f8d-96d8-23228f70983b.png)

If this changed to `' 50px' : '';` the columns are fixed:
![normal](https://user-images.githubusercontent.com/40806497/152420141-faa4c8e2-5bb4-442d-adbc-d35970c4c7ad.png)

Reply:
Oh good, nice catch!
I got confused while interpolating the class after getting the linting error.

Reply:
Linting is known to cause confusion, even with all of the rules set in place it will still allow some oddities through 😀